js/viz/core/center_template.js

```diff
@@ -0,0 +1,86 @@
+
+import { patchFontOptions } from '../core/utils';
+import { noop } from '../../core/utils/common';
+
+const pieChartPlugin = {
+    name: 'center_template_pie_chart',
+    init: noop,
+    dispose: function() {
+        this._centerTemplateGroup.linkOff().dispose();
+    },
+    extenders: {
+        _createHtmlStructure() {
+            this._centerTemplateGroup = this._renderer.g()
+                .attr({ class: 'dxc-hole-template' })
+                .linkOn(this._renderer.root, 'center-template')
+                .css(patchFontOptions(this._themeManager._font))
```

Comment:
It's a good idea to move a parameter function call to a separate variable to simplify a debug.
When u debug u will see which parameter u pass before this chaining and can debug parameter getter separately.
```suggestion
            const patchedFontOptions = patchFontOptions(this._themeManager._font);
            this._centerTemplateGroup = this._renderer.g()
                ...
                .css(patchedFontOptions)
```